sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -33,11 +33,13 @@ import org.apache.spark.sql.execution.command.DDLUtils
 import org.apache.spark.sql.expressions.Window
 import org.apache.spark.sql.hive.test.{TestHive, TestHiveContext}
 import org.apache.spark.sql.types.{DecimalType, StructType}
+import org.apache.spark.tags.ExtendedHiveTest
 import org.apache.spark.util.{ResetSystemProperties, Utils}
 
 /**
  * This suite tests spark-submit with applications using HiveContext.
  */
+@ExtendedHiveTest
```

Comment:
With this tag, the test suite will still run if we do a code change in sql/hive. Right? cc @cloud-fan @vanzin

Reply:
That was the original intent of the tag. I believe originally it ran whenever there was a change to any of the SQL code, but not sure if the module mappings have been updated in the meantime.

Reply:
Thanks!